docker/bacalhau/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM docker:dind as base
```

Comment:
🙈 🙉 🙊

Reply:
OH NO

Reply:
The monkeys are all referring to the lower case `as`, that's all.